core/src/main/java/org/commonjava/maven/ext/core/impl/BaseGroovyManipulator.java

```diff
@@ -94,10 +94,12 @@
                 for ( final String script : scripts )
                 {
                     File found;
-                    if ( script.startsWith( "http" ) || script.startsWith( "file" ))
+                    logger.info( "Processing groovy script {}", script );
+                    if ( script.startsWith( "http:" ) || script.startsWith( "file:" ) )
```

Comment:
Missing `:` here to mark the protocol.

Reply:
I don't see why this is needed?

Reply:
If you want to check if it's a URL, then the URL must start with `scheme + ':'`, or otherwise strings such as `"file.txt"` will match, which is unintended, and it could even match before the the final `else` case.

It also matches what it used elsewhere in the code:

https://github.com/release-engineering/pom-manipulation-ext/blob/8606c8b023b8a60333e41495d3ad38e466254491/core/src/main/java/org/commonjava/maven/ext/core/impl/RepoAndReportingRemovalManipulator.java#L264

https://github.com/release-engineering/pom-manipulation-ext/blob/8606c8b023b8a60333e41495d3ad38e466254491/io/src/main/java/org/commonjava/maven/ext/io/resolver/GalleyInfrastructure.java#L238

https://github.com/release-engineering/pom-manipulation-ext/blob/8606c8b023b8a60333e41495d3ad38e466254491/io/src/main/java/org/commonjava/maven/ext/io/resolver/GalleyInfrastructure.java#L266

https://github.com/release-engineering/pom-manipulation-ext/blob/8606c8b023b8a60333e41495d3ad38e466254491/io/src/main/java/org/commonjava/maven/ext/io/resolver/MavenLocationExpander.java#L115

Reply:
Ok

Reply:
Ok sure, I guess being consistent is sensible.